planning/behavior_path_planner/src/utils/goal_planner/default_fixed_goal_planner.cpp

```diff
@@ -53,11 +134,26 @@ PathWithLaneId DefaultFixedGoalPlanner::modifyPathForSmoothGoalConnection(
       refined_goal = goal;
     }
   }
-
-  auto refined_path = utils::refinePathForGoal(
-    planner_data->parameters.refine_goal_search_radius_range, M_PI * 0.5, path, refined_goal,
-    goal_lane_id);
-
+  double goal_search_radius{planner_data->parameters.refine_goal_search_radius_range};
```

Comment:
@HansOersted 
can you define with const if the value will not be changed

Reply:
made the following change
**const** double range_reduce_by{1}